bin/smoke_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e # Make script exit when a command fail.
+set -u # Exit on usage of undeclared variable.
+# set -x # Trace what gets executed.
+set -o pipefail # Catch failures in pipes.
+
+# Temporarily disable -e mode
+set +e
+for solution in test_data/*/* ; do
+  slug=$(basename $(dirname $solution))
+  # run analysis
+  bin/run.sh $slug $solution $solution
+  # check result
+  bin/check_files.sh $solution
+done
+set -e
```

Comment:
When I mess up with one of the expected results (e.g. change the emoji in `test_data/lasagna/failing_solution/expected_analysis.json`) to simulate a failure, running `./bin/run-tests-in-docker.sh` still has an exit code 0. I think that means the CI won't really report smoke test failures.

Reply:
Oh, good catch. 
I guess using `set +e` in this way is inappropriate.